Documentation/Examples/Cycles.csv

```diff
@@ -0,0 +1,11 @@
+0,SET,LoopVar,10,
+0,SansText,The lines 5 to 7 are the code inside the cycle
+0,SansText,Iterating 10 times,,
+0,JMPZ,8,$LoopVar,
+0,SUB,LoopVar,$LoopVar,1
+0,SansText,Iterations left:,,
+0,SansText,$LoopVar,,
+0,JMPABS,3,,
```

Comment:
I think named labels should be used instead of line numbers, as they allow for lines to be shifted around without ruining the jumps. Line number jumps are left over from a time when labels weren't supported. Labels are supported now, and their use should be encouraged over line numbers.

Reply:
Labels like having variables with the line number?

Reply:
Like this:
`0,:MyLabel,,,
0,JMPABS,MyLabel,,`